tests/ci-helper.test.ts

```diff
@@ -393,6 +393,7 @@ testQ("handle comment allow no name specified (with trailing white space)", asyn
         number: prNumber,
         pullRequestURL: "https://github.com/gitgitgadget/git/pull/59",
         title: "Submit a fun fix",
+        draft: false,
```

Comment:
@hickford did you want to maybe add a test that verifies that a draft PR really is marked as an RFC?

Reply:
Yes but I got stuck and couldn't figure out how.

Might you be interested in finishing this?